tests/manage/mcg/test_namespace_crd.py

```diff
@@ -784,6 +790,148 @@ def test_create_ns_bucket_from_utilized_resources_crd(
         logger.info("Compare between uploaded files and downloaded files")
         assert self.compare_dirs(awscli_pod, amount=3)
 
+    @pytest.mark.polarion_id("OCS-2280")
+    @tier3
+    def test_create_resource_with_invalid_target_bucket_rpc(
+        self,
+        mcg_obj,
+    ):
+        """
+        Test that a proper error message is reported when invalid target
+        bucket is provided during namespace resource creation.
+        """
+        for target_bucket_name in ("", " ", "/*-#$%@^"):
+            nss_obj = NamespaceStore(
+                name="invalid-nss",
+                method="oc",
+                mcg_obj=mcg_obj,
+                uls_name=target_bucket_name,
+            )
+
+            try:
+                nss_obj.verify_health(timeout=15)
+            except AssertionError:
+                logger.info("Invalid NamespaceStore is not healthy as Expected")
+                nss_obj.delete()
+            else:
+                nss_obj.delete()
+                assert False, "Invalid NamespaceStore should not be healthy"
+
+    @pytest.mark.polarion_id("OCS-2282")
+    @tier3
+    def test_delete_resource_used_in_ns_bucket_crd(
+        self, namespace_store_factory, bucket_factory
+    ):
+        """
+        Test that a proper error message is reported when invalid target
+        bucket is provided during namespace resource creation.
+        """
+        logger.info("Create the namespace resources and verify health")
+        nss_tup = ("oc", {"aws": [(1, self.DEFAULT_REGION)]})
+        ns_store1 = namespace_store_factory(*nss_tup)[0]
+        nss_tup = ("oc", {"aws": [(1, self.DEFAULT_REGION)]})
+        ns_store2 = namespace_store_factory(*nss_tup)[0]
+
+        logger.info("Create the namespace bucket on top of the namespace resource")
+        bucketclass_dict = {
+            "interface": "OC",
+            "namespace_policy_dict": {
+                "type": "Multi",
+                "namespacestores": [ns_store1, ns_store2],
+            },
+        }
+        bucket_factory(
+            amount=1,
+            interface=bucketclass_dict["interface"],
```

Comment:
Amount is set to be 1 by default, and the interface is automatically pulled from the `bucketclass_dict` if one is given, so both can be removed